ci/cppcheck/cppcheck-new-issues.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-scriptdir="$(cd "${0%/*}" && pwd)"
-rootdir="${scriptdir%/*/*}"
+scriptdir=$(dirname "$(readlink -f "$0")")
+rootdir=$(realpath "$scriptdir/../..")
 
-# shellcheck source=../../tools/functions.sh
+# shellcheck source=tools/functions.sh
```

Comment:
The shellcheck invocation that is used inside an IDE relies on a path that is relative to the script itself, not to the root directory.

Fortunately, you can simply use `shellcheck -P SCRIPTDIR` to allow paths relative to the script.

Reply:
Agree.
Fixed used shellcheck command.